config/locales/views/campagne_mailer.yml

```diff
@@ -0,0 +1,18 @@
+fr:
+  campagne_mailer:
+    relance:
+      objet: "%{prenom}, quelques conseils pour démarrer sur eva - Positionnement des compétences"
+      intro: |
+        ### Bonjour %{prenom},
+
+        Il y a 30 jours vous avez créé un compte sur le service numérique eva. Vous vouliez certainement accompagner différemment vos bénéficiaires. Nous vous rappelons qu'eva permet de détecter l'illettrisme et de valoriser vos "%{cible_evaluation}" au travers des compétences transversales acquises.
+      x_personnes_utilisent_eva:
```

Comment:
Il manque une clé `one`
`can not be used with :count => 1. key 'one' is missing.v`